experiments/standalone/Vegetation/timestep_test.jl

```diff
@@ -0,0 +1,249 @@
+    hydraulics = plant_hydraulics,
+    soil_driver = soil_driver,
+    atmos = atmos,
+    radiation = radiation,
+);
+
+
+Y, p, coords = ClimaLand.initialize(canopy)
+exp_tendency! = make_exp_tendency(canopy)
+imp_tendency! = make_imp_tendency(canopy)
+jacobian! = make_jacobian(canopy);
+jac_kwargs =
+    (; jac_prototype = ClimaLand.ImplicitEquationJacobian(Y), Wfact = jacobian!);
+
+
+ψ_leaf_0 = FT(-2e5 / 9800)
+ψ_stem_0 = FT(-1e5 / 9800)
+
+S_l_ini =
+    inverse_water_retention_curve.(
+        retention_model,
+        [ψ_stem_0, ψ_leaf_0],
+        ν,
+        S_s,
+    )
+
+Y.canopy.hydraulics.ϑ_l.:1 .= augmented_liquid_fraction.(ν, S_l_ini[1])
+Y.canopy.hydraulics.ϑ_l.:2 .= augmented_liquid_fraction.(ν, S_l_ini[2])
+
+
+seconds_per_day = 3600 * 24.0
+t0 = 150seconds_per_day
+N_days = 100
+tf = t0 + N_days * seconds_per_day
+evaluate!(Y.canopy.energy.T, atmos.T, t0)
+set_initial_cache! = make_set_initial_cache(canopy)
+set_initial_cache!(p, Y, t0);
+
+saveat = Array(t0:(3 * 3600):tf)
+updateat = Array(t0:(3600 * 3):tf)
+drivers = ClimaLand.get_drivers(canopy)
+updatefunc = ClimaLand.make_update_drivers(drivers)
+cb = ClimaLand.DriverUpdateCallback(updateat, updatefunc)
+
+timestepper = CTS.ARS111();
+ode_algo = CTS.IMEXAlgorithm(
+    timestepper,
+    CTS.NewtonsMethod(
+        max_iters = 6,
+        update_j = CTS.UpdateEvery(CTS.NewNewtonIteration),
+    ),
+);
+
+prob = SciMLBase.ODEProblem(
+    CTS.ClimaODEFunction(
+        T_exp! = exp_tendency!,
+        T_imp! = SciMLBase.ODEFunction(imp_tendency!; jac_kwargs...),
+        dss! = ClimaLand.dss!,
+    ),
+    Y,
+    (t0, tf),
+    p,
+);
+
+ref_dt = 6.0
+ref_sol =
+    SciMLBase.solve(prob, ode_algo; dt = ref_dt, callback = cb, saveat = saveat);
+ref_T = [parent(ref_sol.u[k].canopy.energy.T)[1] for k in 1:length(ref_sol.t)]
+mean_err = []
+p95_err = []
+p99_err = []
+dts = [12.0, 24.0, 48.0, 100.0, 225.0, 450.0, 900.0, 1800.0, 3600.0]
+for dt in dts
+    @info dt
+    saveat = Array(t0:(3 * 3600):tf)
+    evaluate!(Y.canopy.energy.T, atmos.T, t0)
+    updateat = Array(t0:(3600 * 3):tf)
+    updatefunc = ClimaLand.make_update_drivers(drivers)
+    cb = ClimaLand.DriverUpdateCallback(updateat, updatefunc)
+
+    @time sol =
+        SciMLBase.solve(prob, ode_algo; dt = dt, callback = cb, saveat = saveat)
+    T = [parent(sol.u[k].canopy.energy.T)[1] for k in 1:length(sol.t)]
+    ΔT = abs.(T .- ref_T)
+    push!(mean_err, mean(ΔT))
+    push!(p95_err, percentile(ΔT, 95))
+    push!(p99_err, percentile(ΔT, 99))
+end
+savedir = joinpath(pkgdir(ClimaLand), "experiments/standalone/Vegetation");
+fig = Figure()
+ax = Axis(
+    fig[1, 1],
+    xlabel = "Time (minutes)",
+    ylabel = "Temperature (K)",
+    xscale = log10,
+    yscale = log10,
+)
+dts = dts ./ 60
+lines!(ax, dts, FT.(mean_err), label = "Mean Error")
+lines!(ax, dts, FT.(p95_err), label = "95th% Error")
+lines!(ax, dts, FT.(p99_err), label = "99th% Error")
+axislegend(ax)
+save(joinpath(savedir, "errors.png"), fig)
```

Comment:
The mean error plot currently on Buildkite indicates a convergence order much lower than 1, which is the value I would expect to see for IMEX Euler (`ARS111`). This might be evidence of a non-negligible issue with the Jacobian approximation, but it could also be an issue with the test setup. In order to measure reasonable convergence orders in `ClimaTimeSteppers.jl`, I needed to use a reference `dt` that was several orders of magnitude smaller than the tested values of `dt`, even for simple nonlinear problems. It would be good to rerun this test with smaller `dt`s, particularly with a smaller reference `dt`, to see whether you can get closer to the expected convergence order.